### ISSUE TITLE
1.3 Adaptable

### ISSUE THREAD

**Issue by DonnaDuiker** [alkem-io/client-web#8369]
# Background
After going through the WCAG 2.1 checklist, creating issues for all A and AA points that still need work. 1 issue for each guideline.

This is guideline 1.3 Adaptable

# Acceptance Criteria

- [ ] 1.3.1 Info and Relationships: Semantic HTML is used (headings, lists, tables). (More work needed: some divs for buttons. Not using headings correctly (not using H1, but using H2 for space titles)
- [ ] 1.3.5 Identify Input Purpose: Input fields have machine-readable labels (e.g., 'autocomplete' ). (More work needed	We should check if it is still the case. Labelname. Comments and search are missing according to audit)

# Context

[Excel file](https://stichtingcherrytwist.sharepoint.com/:x:/s/Product/EawxIsWzBaRHt8K2E1tUwmsBzd93mWRRiImOP2M0muxujQ?e=eYV2Qr)

[W3C website](https://www.w3.org/TR/WCAG21/)
